src/content/tutorials/connectivity/ingress/configuration/index.md

```diff
@@ -0,0 +1,640 @@
+            name: SERVICE_NAME
+            port:
+              number: SERVICE_PORT
+```
+
+Make sure to use the exact annotation scheme `nginx.ingress.kubernetes.io/configuration-snippet` in the `metadata` section of the manifest.
+
+In case you want to set up a general HTTP snippet you can define it at [ingress nginx controller ConfigMap](#configmap) level.
+
+## Global (per cluster) options {#configmap}
+
+Your Giant Swarm installation comes with a default configuration for the ingress controller.
+
+You can override these defaults by setting your per cluster configuration in the form of a ConfigMap named `ingress-nginx-user-values` in the management cluster.
+
+The page [App configuration reference]({{< relref "/vintage/getting-started/app-platform/app-configuration/index.md" >}}) contains more information how to set user defined configuration for the ingress nginx controller.
+
+### Where's the user values ConfigMap
+
+Given the cluster you are trying to configure has id: `123ab`
+
+You will find the `ingress-nginx-user-values` ConfigMap on the management cluster in the `123ab` namespace:
+
+```nohighlight
+$ kubectl -n 123ab get configmap ingress-nginx-user-values
+NAME                        DATA      AGE
+ingress-nginx-user-values   0         11m
+```
+
+__Warning__:
+
+Please don't edit any of the other ingress nginx controller related ConfigMaps.
+
+Only the user ConfigMap is safe to edit.
+
+---
+
+### How to set configuration options using the user values ConfigMap
+
+You are able to set any value from the [upstream documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/) by including them in the user values ConfigMap under the `data.values` field like so:
+
+```yaml
+# On the management cluster
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingress-nginx-user-values
+  namespace: NAMESPACE
+data:
+  values: |
+    controller:
+      config:
+        log-format-upstream: "MY EDITED LOG FORMAT - $status $body_bytes_sent $http_referer"
+```
+
+However keep in mind: With great power comes great responsibility!
+
+If the ConfigMap doesn't exist, create it. In this case you'll need to reference it in the App CR of the ingress controller.
+
+```yaml
+# Add missing keys to the spec field of the App CR
+spec:
+  userConfig:
+    configMap:
+      name: ingress-nginx-user-values
+      namespace: NAMESPACE
+```
+
+Any defaults that we override are visible in the following `values.yaml` file, under the `controller.config` key. Check this [`values.yaml`](https://github.com/giantswarm/ingress-nginx-app/blob/main/helm/ingress-nginx/values.yaml) as an example.
+
+Don't copy all the defaults if you don't need to change them, that way we can adjust them in case they need to change.
+
+Do make sure you look at the right tag of that repository, when reading this file check that the tag corresponds to the version of the ingress nginx controller running on your cluster.
+
+### Configure Proxy Protocol
```

Comment:
```suggestion
Any defaults we override are visible in the following `values.yaml` file under the `controller.config` key. Check this [`values.yaml`](https://github.com/giantswarm/ingress-nginx-app/blob/main/helm/ingress-nginx/values.yaml) as an example.

Don't copy all the defaults if you don't need to change them; that way, we can adjust them in case they need to be changed.

Please make sure you look at the right tag in that repository. When reading this file, check that the tag corresponds to the version of the ingress nginx controller running on your cluster.

### Configure proxy protocol
```